roles/branch-validation/tasks/main.yml

```diff
@@ -92,6 +82,17 @@
     delay: 5
     timeout: 300
     host: "{{ fip.floating_ip.floating_ip_address }}"
+  when: not local_wait_for
+
+- name: wait for instances to be reachable
+  sudo: no
+  wait_for:
+    port: 22
+    delay: 5
+    timeout: 300
+    host: "{{ fip.floating_ip.floating_ip_address }}"
+  delegate_to: localhost
+  when: local_wait_for
```

Comment:
there were occasions in my testing that remote wait failed because of firewall configuration (rainier) and offering a quick workaround seemed prudent.